CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contributing
+
+When contributing to this repository, the changes will come under the following categories:
+
+1. Data Update:
+   If it's your personal identification information, please create a PR without creating an
+   issue for it.
+   If it's regarding an event, kindly create a new issue to discuss it first before making a
+   PR.
+2. Layout Update:
+   These changes also must be accompanied with an issue. First discuss it with someone by
+   creating an issue and then go on to make a PR. Also do not work on the PR unless assigned
+   to it.
+
+For any kind of changes, we have a detailed document describing the process at 
```

Comment:
Specify what's available where, i.e.
> For updating data and details of pushing changes, we have a detailed document ...

And that the code structure info is here